flow/client_server/static/internal/groups/create/js/script.js

```diff
@@ -0,0 +1,36 @@
+function onceDocumentReady() {
+	getURLs(startSubmitButtonListener);
+}
+
+function startSubmitButtonListener() {
+	$('#submit').click(function () {
+		var group_name = $('#group_name').val();
+		var aid = getAID();
+		var data = {
+			group_name: group_name,
+			aid: aid
```

Comment:
Please, I would love to know. What does aid:aid mean or do?
     Sincerely, Camden Accuosti